src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -99,6 +99,7 @@ public class BlockGUI extends ModElementGUI<Block> {
 	private ProcedureSelector onRedstoneOff;
 	private ProcedureSelector onHitByProjectile;
 	private ProcedureSelector onBonemealSuccess;
+	private ProcedureSelector onEntityFallsOn;

```

Comment:
Please move this into another PR, it can be PR that we merge before this one, but this event isn't directly correlated to the bouncy blocks directly. Still very useful trigger, just should be in separate PR